apps/web/src/services/asset-inventory/collector/modules/CollectorNoData.vue

```diff
@@ -0,0 +1,41 @@
+<template>
+    <p-empty
+        show-image
+        show-button
+        image-size="md"
+    >
+        <template #image>
+            <img alt="empty-image"
+                 src="@/assets/images/illust_microscope.svg"
+            >
+        </template>
+        <template #button>
+            <p-button
+                icon-left="ic_plus_bold"
+                @click="handleCreate"
+            >
+                <span>Create Collector</span>
+            </p-button>
+        </template>
+        There’s no collector created yet.
+    </p-empty>
```

Comment:
nitpick: This is really a small, nitpick comment just to share the tips 💗
I usually put the translation codes first since it is not efficient can cause human mistakes to change all the hard coded languages into translation codes at once after implementing all the things.
But this is just my style, so I just wanted to note it to you to prevent my cases.😄